### DISCUSSION TITLE
eclipse 2024-09 authorization failed. please help me

### DISCUSSION THREAD

**Discussion by changxeokSong** [Genuitec/Copilot4Eclipse-community#24] [Help]
I’m trying to login to GitHub Copilot and I get this error.
I want you to tell me how to solve it

Version: 2024-09 (4.33.0)
Build id: 20240905-0614
2021, Apple M1 Pro, 15.0 beta(24A5327a)

However, It was worked normally when it was Windows 10
please help me.

there are error
————————————————————————————————————————————
Authorization request failed: process aborted. Please try again to resume use of GitHub Copilot features.
————————————————————————————————————————————
<img width="581" alt="11" src="https://github.com/user-attachments/assets/d30d4c48-64c3-489f-a023-31e82e4c4017">

**Comment by wayneparrott** [comment]
A few things to try:

0. From a browser logout of GitHub and then log back in. This will update your GitHub access token. Then try logging in from Copilot4Eclipse.

1. Since you share that Copilot4Eclipse worked on your Windows machine, verify that your proxy settings are equivalent - I am assuming you are operating on the same network.

2. Please review the Eclipse log (select from Eclipse Window menu > Show View > General > Error Log) for relevant error information. Please share any details.

Wayne
Copilot4Eclipse Team

**Reply by changxeokSong** [reply]
I've tried everything given and it still doesn't work.

and

In a Windows environment, the network connections window looks like this

![KakaoTalk_Photo_2024-09-25-09-44-33](https://github.com/user-attachments/assets/970e72b8-ea45-4cea-b9bc-8a7008707af7)

and in a Mac, it looks like this.
<img width="907" alt="스크린샷 2024-09-25 오전 9 44 02" src="https://github.com/user-attachments/assets/a0cb4902-f086-40e5-bb19-7984936a675c">

The other settings are all in the same condition.
But, I couldn't specify 'dynamic' in the settings.


I will attach a log error.

<img width="1047" alt="스크린샷 2024-09-25 오전 9 38 49" src="https://github.com/user-attachments/assets/600d1324-002f-4568-826a-6a3ef5fbe934">
<img width="1070" alt="스크린샷 2024-09-25 오전 9 37 32" src="https://github.com/user-attachments/assets/f552a2d9-9d8c-4be1-8903-cd6ab6d40c7d">
